templates/dashboard.html

```diff
@@ -229,7 +229,7 @@
                   </div>
                 </span>
                 <div style="font-weight:300;font-size:1rem;display:flex;justify-content:center" class="pb-2">
-                  <span id="dashboard-info">Found 0 pending, 0 active and 0 exited validators</span>
+                  <span id="dashboard-info">0 active (0 online, 0 offline), 0 pending, 0 exited validators</span>
```

Comment:
Afaik this is just a dummy text that just serves to illustrate what the formatting looks like.

Reply:
This is not entirely true, this text is shown as long as the dashboard is loading. I surfed to the link you sent to test your changes and saw said text for ~2s.

Reply:
I see, thanks for the info.